gm4_crossbow_cartridges/data/gm4_crossbow_cartridges/functions/projectile/redstone_torch/check.mcfunction

```diff
@@ -0,0 +1,19 @@
+# Check blocks, to place or drop redstone torch
+# @s = arrow with tag gm4_cb_rtorch, in a block
+# at @s
+# run from projectile/redstone_torch/loop
+
+# summon torch item if torch already exists at this block
+execute if block ~ ~ ~ #gm4_crossbow_cartridges:torch run loot spawn ~ ~ ~ loot gm4_crossbow_cartridges:redstone_torch
```

Comment:
why do these use custom loot tables? You can just reference the minecraft block loot table for the torch types. Example below
```suggestion
execute if block ~ ~ ~ #gm4_crossbow_cartridges:torch run loot spawn ~ ~ ~ loot minecraft:blocks/redstone_torch
```

Reply:
Good question, didn't actually know about vanilla minecraft loot tables for some reason. Same problem for the saddle in Chairs then

Reply:
Actually no, not the same for Chairs. This one is a block loot table, meaning when a player breaks a torch this loot table will drop (which is consistent with what would happen here). There's no block loot table for saddles cuz, well it's not a block. I'm taking care of that tho, so don't worry about chairs

Reply:
Yep just looked into it, I didn't make the connection that the torches are blocks so just repeated what I did for chairs, explains why I overlooked these minecraft loot tables